peachjam_ml/models.py

```diff
@@ -147,6 +147,50 @@ def refresh_for_document(cls, document):
 
         return doc_embedding
 
+    @classmethod
+    def get_average_embedding(cls, pks):
+        """Get the average embedding for a set of documents."""
+        if not pks:
+            return None
+
+        if len(pks) == 1:
```

Comment:
This is a premature optimisation. If there's only one, the db will handle it during Avg() etc

Reply:
i think the more common case is the document detail view which will have only one embedding. we don't need to be calculating the average for one document each time we call that view.

Reply:
It's not expensive to calculate the average of one, and it's all done in the DB. You can safely delete 6 lines of code -- less to test, less to go wrong.